src/handler.ts

```diff
@@ -30,10 +34,25 @@ export async function handlePullRequest (context: Context): Promise<void> {
     return
   }
 
-  const reviewers = chooseUsers(owner, config.reviewers, config.numberOfReviewers)
+  if(config.useReviewGroups && !config.reviewGroups){
+    throw new Error('no review group variable defined in the configuration file')
+    return;
+  }
 
-  let result: any
+  if(config.useAssigneeGroups && !config.assigneeGroups){
+    throw new Error('no assignee group variable defined in the configuration file')
+    return;
+  }
 
+  let reviewers: string[] = []
+  if(config.useReviewGroups && Object.keys(config.reviewGroups).length > 0) {
+    
+    reviewers = selectUsersFromGroups(owner, config.reviewGroups, config.numberOfReviewers)
+  } else if(config.reviewers && (config.addReviewers || config.addAssignees)) { 
+    reviewers = chooseUsers(owner, config.reviewers, config.numberOfReviewers)
```

Comment:
Here we use choose, for groups we use select. Could we make it consistent?

Reply:
Done.